egs2/libritts_r/enh1/local/resample_wav_scp.py

```
@@ -0,0 +1,90 @@
+def _resample_one(utt, src, wav_dir, target_sr):
+    out_path = Path(wav_dir) / f"{utt}.wav"
+    if out_path.exists():
+        return utt, str(out_path), None
+
+    try:
+        # pipe command or plain path — use sox for both
+        if src.endswith("|"):
+            # e.g. "sox /path/to/file.wav -r 24000 -t wav - |"
+            cmd = src.rstrip(" |").split()
+            # replace or add -r target_sr
+            result = subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)